src/scripts/shell/platform-choice/index.js

```diff
@@ -0,0 +1 @@
+export * from './platform-choice.component';
```

Comment:
Makes sense. Perhaps a bit overkill for this simple component, but I guess there could eventually be more here.